packages/manager/src/features/Profile/SecretTokenDialog/SecretTokenDialog.tsx

```diff
@@ -46,7 +49,11 @@ export const SecretTokenDialog: React.FC<CombinedProps> = (props) => {
   const classes = useStyles();
   const { title, value, objectStorageKey, open, onClose } = props;
 
-  const actions = renderActions(onClose);
+  const modalConfirmationButtonText = objectStorageKey
+    ? `I Have Saved My Secret Key`
```

Comment:
This doesn't need to be in back ticks.
```suggestion
    ? 'I Have Saved My Secret Key'
```